docs/reference/ml/anomaly-detection/ml-configuring-transform.asciidoc

```diff
@@ -1,14 +1,14 @@
 [role="xpack"]
 [[ml-configuring-transform]]
-= Transforming data with script fields
+= Altering data with runtime mappings
 
-If you use {dfeeds}, you can add scripts to transform your data before
-it is analyzed. {dfeeds-cap} contain an optional `script_fields` property, where
-you can specify scripts that evaluate custom expressions and return script
-fields.
+If you use {dfeeds}, you can use runtime mappings to alter your data before it 
```

Comment:
If I'm understanding correctly, "runtime mappings" are just the method runtime fields are implemented in datafeeds. In which case, I think we should refer to the bigger concept of runtime fields as much as possible. For example:

```suggestion
If you use {dfeeds}, you can use runtime fields to alter your data before it 
```